CONTRIBUTING.md

```diff
@@ -0,0 +1,40 @@
+# Contributing to Stencil
```

Comment:
Should we add any Licensing information like "By contributing to Stencil, you agree that your contributions will be licensed under its MIT license."